pkg/daemon/ceph/client/controller_utils.go

```diff
@@ -0,0 +1,78 @@
+/*
```

Comment:
What about adding this helper under `pkg/operator/ceph/spec`? That package is really a collection of helpers for controllers. And how about renaming that package to `pkg/operator/ceph/controller`?

Reply:
Sounds good.